spec/features/admin/users/unauthorized_user_tries_to_view_a_list_of_users_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'Unauthorized user tries to view a list of users' do
+  context 'when moderator try to access the admin/users' do
```

Comment:
May I suggest describing this as `when the user is a moderator`?

The feature is stating that you are trying to view a list of users.